addons/digest/data/digest_tips_data.xml

```diff
@@ -69,6 +69,42 @@
     <p class="tip_content">When following documents, use the pencil icon to fine-tune the information you want to receive.
 Follow a project / sales team to keep track of this project's tasks / this team's opportunities.</p>
     <img src="https://download.odoocdn.com/digests/digest/static/src/img/milk-following.png" width="540" class="illustration_border" />
+</div>
+            </field>
+        </record>
+        <record id="digest_tip_digest_5" model="digest.tip">
+            <field name="name">Tip: Join the Dark Side</field>
+            <field name="sequence">3300</field>
+            <field name="group_id" ref="base.group_user"/>
+            <field name="tip_description" type="html">
+<div>
+    <p class="tip_title">Tip: Join the Dark Side</p>
+    <p class="tip_content">Feeling eye strain? Give your eyes a break by switching to Dark Mode.</p>
+    <img src="/digest/static/src/img-v17/switch-theme.gif" width="540" class="illustration_border"/>
+</div>
+            </field>
+        </record>
+        <record id="digest_tip_digest_6" model="digest.tip">
+            <field name="name">Tip: Your Business right in their pocket</field>
+            <field name="sequence">3400</field>
+            <field name="group_id" ref="base.group_system"/>
+            <field name="tip_description" type="html">
+<div>
+    <p class="tip_title">Tip: Your Business right in their pocket</p>
+    <p class="tip_content">Connect your WhatsApp account to Odoo to receive inbound messages directly in your CRM.</p>
```

Comment:
This one requires enterprise code, hence I suggest to not have it in community (I know, marketing blabla, but quand même :D ) .